Makefile

```diff
@@ -52,16 +52,15 @@ format: # format go code
 	gofmt -s -w ./
 
 .PHONY: docker/server
-docker/server:
+docker/server: docker/odr
```

Comment:
I think this should be reverted back, right?

Reply:
Nope, we want to build them both.

Reply:
Hm, it'd be good to have a separate server make target at least for when you're doing server dev work and only want to rebuild that. Maybe we have a `docker/all` that's this versus `docker/server` which is just for the server? or do we need to always build the ondemand runner Dockerfile for any changes in the server?

Reply:
The issue is that you need both images to do local work since we create an ondemand runner by default.

Reply:
Suggested in slack that we could have a `docker/server-only` target. We wanna make sure peoples workflows don't break if they're used to `docker/server` giving everything they need so that's the main reason for the change.